### DISCUSSION TITLE
DELETE /api/v1/chatmessage/ not working

### DISCUSSION THREAD

**Discussion by younessouilmi** [FlowiseAI/Flowise#1519] [Q&A]
Hy guys,

I am struggling with a persistent problem with the Message API

GET /api/v1/chatmessage/{my-chatflowid} returns several hundred messages
DELETE /api/v1/chatmessage/{my-chatflowid} returns {"raw":[],"affected":0}
a subsequent GET /api/v1/chatmessage/{my-chatflowid} confirms that messages have not been deleted

am I miss-using the DELETE endpoint? 

Thanks in advance

**Comment by HenryHengZJ** [comment]
Bug fixed - https://github.com/FlowiseAI/Flowise/pull/1525
